### DISCUSSION TITLE
Error: invalid Bech32 prefix; expected crocncl, got crocnclcons

### DISCUSSION THREAD

**Discussion by djellsy** [crypto-org-chain/chain-main#465] [Q&A] [answered]
Successfully created a validator and initial stake:
chain-maind tx staking create-validator \
--from="cro160k8wu30h4kwnkm2qtsd5wn788gk04wc80s4n3" \
--amount="1000cro" \
--pubkey=<my chain-maind tendermint show-validator address>  \
--moniker="aiace" \
--security-contact="<my email>/email" \
--chain-id="crypto-org-chain-mainnet-1" \
--commission-rate="0.09" \
--commission-max-rate="0.20" \
--commission-max-change-rate="0.01" \
--min-self-delegation="1" \
--gas 8000000 \
--gas-prices 0.1basecro

When I try to delegate additional funds I get the error: 
Error: invalid Bech32 prefix; expected crocncl, got crocnclcons

Additionally, if I try to redelegate funds using chain-maind tx redelegate I get an error:
Error: empty delegator address

I feel like something went screwy. I can't add or move additional funds. Any ideas?

**Comment by lezzokafka** [comment, marked as answer]
Hello @djellsy, kindly use the `validator address` ( starting with `crocncl...` )when doing delegate/redelegate under the usage of 

`chain-maind tx staking delegate [validator-addr] [amount] [flags]` 

For example, `crocncl160k8wu30h4kwnkm2qtsd5wn788gk04wcyznu3d` will be the validator-addr of your [node](https://crypto.org/explorer/validator/crocncl160k8wu30h4kwnkm2qtsd5wn788gk04wcyznu3d).

**Comment by djellsy** [comment]
Hi Lezzo. Yes, I am using:
chain-maind tx staking delegate crocncl160k8wu30h4kwnkm2qtsd5wn788gk04wcyznu3d 1000cro
and I get the error:
Error: empty delegator address

Thanks!

**Comment by djellsy** [comment]
I can try
chain-maind query staking delegation crocncl160k8wu30h4kwnkm2qtsd5wn788gk04wcyznu3d crocnclcons1vmf9ars6p0kmgx0pld2e4kz7m825um89dlh64q
and get the same error: 
Error: empty delegator address

**Comment by djellsy** [comment]
Updated chain-maind to version 1.2 and now when running:
chain-maind query staking delegation crocncl160k8wu30h4kwnkm2qtsd5wn788gk04wcyznu3d crocnclcons1vmf9ars6p0kmgx0pld2e4kz7m825um89dlh64q
I am back to the original error:
Error: invalid Bech32 prefix; expected cro, got crocncl
